### DISCUSSION TITLE
Moet iedere organisaite ene eigen catalogi runnen?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#94] [Q&A]
of accepteren we het ook als meerder organisatie samen een catalogi runnen?

Aka zou ik ook moeten kunnen inlogen en identificeren via de landelijke catalogi

**Comment by RonaldvCortenberghe** [comment]
Ja, iedere organisatie **moet** zijn eigen catalogi kunnen runnen, maar dat is geen verplichting. Ik kan me zomaar voorstellen dat enkele organisaties een gezamenlijke catalogi willen gebruiken (uit kostenoverweging bijv.). Maar dat is dan niet per definitie een landelijke catalogi. Dat vraagt denk ik wel wat van de oplossing omdat het gebruik van een component door een of meerdere organisaties binnen de ene catalogi moet worden opgelost
